.travis.yml

```diff
@@ -40,10 +40,7 @@ before_script:
 
 script:
   - cargo build --verbose --features llvm_stable
-  - make test
-  - git add -A
-  - git diff @
```

Comment:
I'd also keep the `diff` and `diff-index` checks, since they're harmless, but avoid potential errors.